news/_posts/translation/2020-07-10-do-you-speak-prestashop-june-2020.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "Do you speak PrestaShop? – June 2020 edition"
+subtitle: "Where you'll learn that Esperanto is on the rise"
+date:  2020-07-10 14:30:11
+authors: [ JuneByun ]
+icon: icon-world
+image: /assets/images/2019/01/Build-Crowdin-banner.jpg
+tags: [translation, i18n, l10n]
+---
+
+Contributing to PrestaShop is not only about the code, it's also about taking part in the [PrestaShop translation project](https://crowdin.com/project/prestashop-official)! This report will tell you how the translations of the software evolved in May.
+
+![Crowdin Monthly banner](/assets/images/2019/01/Build-Crowdin-banner.jpg)
+
+## Project news
+
+
+### First news, Esperanto is on the rise :fire:
+
+I am thrilled to let you know that Esperanto has been on the rise since the last past months. From the beginning of May to the end of June, the Esperanto translation project went from 9% to 34% words translated, that is to say a 25% evolution. I would like to take this opportunity to thank [mirmal](https://crowdin.com/profile/mirmal) and [Milan Kolka](https://crowdin.com/profile/milankolka) for your work! :heart_eyes:
+
+### Interested in becoming a proofreader?
+
+A lot of translators dedicate their time to contribute to  the PrestaShop projects on Crowdin but some strings cannot reach the software because they remain unvalidated. This is for instance the case of the Esperanto project which, as mentioned above, had an increase of translated words but no progress in approving these strings.
+
+I thus wanted to tell you that you can ask me if you wish to become a proofreader. If you are already an active translator and wish to be more involved in the translation project by vouching for the translation consistency and approving strings, you can [reach out to me](https://crowdin.com/profile/june.byun)! :wink:
```

Comment:
@LouiseBonnard 'hope it's ok now! :)

Reply:
```suggestion
### Interested in becoming a proofreader?

A lot of translators dedicate their time to contribute to the PrestaShop's translation projects on Crowdin but some strings cannot reach the software because they remain unvalidated. This is for instance the case of the Esperanto project which, as mentioned above, had an increase of translated words but no progress in approving these strings.

I thus wanted to tell you that you can ask me if you wish to become a proofreader. If you are already an active translator and wish to be more involved in the translation project by vouching for the translation consistency and approving strings, you can [reach out to me](https://crowdin.com/profile/june.byun)! :wink:
```

Reply:
Perfect, thank you very much! I've just added the word 'translation' not to confuse the reader with the PrestaShop project like the open source project on GitHub. ;-)

Reply:
Hoorayyyy \o/ Thank you for proofreading it! :)
I'll take the WIP label off and let Antoine know that it is ready to be published tomorrow!